### DISCUSSION TITLE
Welcome to AstakCodex Discussions!

### DISCUSSION THREAD

**Discussion by AstakCodex** [AstakCodex/AstakCodex#3] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JayantGoel001** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/54479676/173169794-33974338-8653-4cc7-be12-187441cfccd6.png)
